eth/v1alpha1/validator.proto

```diff
@@ -310,6 +313,32 @@ message ValidatorStatusResponse {
 
     // The position in the activation queue of pending validators.
     uint64 position_in_activation_queue = 5;
+
+    // A list of validator status metadata mapped 1-to-1
+    // with the public keys in the request.
+    repeated ValidatorStatusMetadata statuses = 6;
+    message ValidatorStatusMetadata {
```

Comment:
This construction seems odd. 

For example, there's already `eth1_deposit_block_number`, `deposit_inclusion_slot`... etc in `ValidatorStatusResponse` which represents a single validator status. But here we add a repeated `ValidatorStatusMetadata` to represent multiple validator status within `ValidatorStatusResponse `

I'm worry about the potential miss use here.

Reply:
Yes. I was worried about this too in my previous comment. However, I am not sure if we can deprecate the old fields b/c they're used here:
https://github.com/prysmaticlabs/ethereumapis/blob/df460bd3d84be4ff3df0658395c7dc9d2a7e7b3d/eth/v1alpha1/validator.proto#L239

The alternative solution I offered was to create a temporary endpoint for [multipleValidatorStatus](https://github.com/prysmaticlabs/ethereumapis/pull/148/commits/e5ac6347f8660769bce86a9adf3951a6778c7e14). And we can deprecate the entire `multipleValidatorStatus` endpoint in a refactor later on?

Reply:
I think `MultipleValidatorStatus` is a good compromise here. What do you think? @rauljordan